modules/date.js

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+const setDate = () => {
+  const datehandler = document.querySelector('.date');
+  datehandler.innerHTML += `<h2 class="dates">${DateTime.now().toISO()}<h2>`;
+};
```

Comment:
- [ ] [OPTIONAL] Kindly it would be nice if you display the updated using `setInterval` it will look better.

```javascript
const setDate = () => {
  const datehandler = document.querySelector('.date');
  setInterval(() => {
    datehandler.innerHTML = `<h2 class="dates">${DateTime.now().toISO()}<h2>`;
  }, 1);
};
```